README.md

```diff
@@ -364,3 +364,51 @@ When modifying the chart:
 ## License
 
 This project is licensed under the terms specified in the LICENSE file.
+Once these prerequisites are met, you can open this folder in VS Code and use the "Reopen in Container" command to launch the environment.
```

Comment:
We already have instructions about the dev container above, I don't think we need more of them here.